docs/configuration.md

```diff
@@ -267,11 +267,10 @@ There are a few exceptions to these rules:
 1. If a configuration file is passed directly via `--config`, those settings are used for _all_
     analyzed files, and any relative paths in that configuration file (like `exclude` globs or
     `src` paths) are resolved relative to the _current_ working directory.
-1. If no config file is found in the filesystem hierarchy, Ruff will fall back to using
-    a default configuration. If a user-specific configuration file exists
-    at `${config_dir}/ruff/pyproject.toml`, that file will be used instead of the default
-    configuration, with `${config_dir}` being determined via [`etcetera`'s native strategy](https://docs.rs/etcetera/latest/etcetera/#native-strategy),
-    and all relative paths being again resolved relative to the _current working directory_.
```

Comment:
I find the new text slightly less clear because it omits what happens if no configuration file is found at all (including the user configuration)

Reply:
I found the "Ruff's default settings are equivalent to:" part sufficient on this, but indeed it can be good to restate it here, I'll tweak it asap